cms/models.py

```diff
@@ -934,9 +969,20 @@ def get_context(self, request, *args, **kwargs):
         # Hits a circular import at the top of the module
         from courses.models import CourseRunEnrollment
 
+        self.course = (
+            Course.objects.filter(id=self.course_id)
+            .select_related("program", "program__programpage")
+            .prefetch_related(
+                "courseruns",
+                Prefetch(
+                    "courseruns__products", Product.objects.with_ordered_versions()
+                ),
+            )
+            .first()
+        )
```

Comment:
The reason for doing this is to do the select related. We do not need this for programs.